src/adt/image.rs

```diff
@@ -36,33 +34,60 @@ impl PositionOptionT for Image {
 #[typetag::serde]
 impl SvgTangibleObject for Image {
     fn to_svg(&self, size: Size, position: Position) -> (Element, Option<Element>) {
-        let mut element = Element::new("image");
-        element.set_attr("width", size.0.to_string());
-        element.set_attr("height", size.1.to_string());
-        element.set_attr("x", position.0.to_string());
-        element.set_attr("y", position.1.to_string());
-        element.set_attr("href", self.path.clone());
-
-        (element, None)
+        let mut svg = Element::new("svg");
+        svg.set_attr("width", size.0.to_string());
+        svg.set_attr("height", size.1.to_string());
+        svg.set_attr("x", position.0.to_string());
+        svg.set_attr("y", position.1.to_string());
+
+        let mut defs = Element::new("defs");
+        let mut clip_path = Element::new("clipPath");
+        clip_path.set_attr("id", "clip");
+
+        let mut rect = Element::new("rect");
+        rect.set_attr("width", "100%");
+        rect.set_attr("height", "100%");
+        rect.set_attr("rx", self.round.unwrap_or(0).to_string());
+
+        clip_path.append_child(rect);
+
+        defs.append_child(clip_path);
+        svg.append_child(defs);
+        let mut img = Element::new("image");
+        img.set_attr("href", self.path.clone());
+        img.set_attr("width", "100%");
+        img.set_attr("height", "100%");
+        img.set_attr("clip-path", "url(#clip)");
```

Comment:
If we have multiple images, the ID `#clip` would need to be unique for each one of them. I am going to create a follow-up ticket (FL-24).

Reply:
> If we have multiple images, the ID `#clip` would need to be unique for each one of them. I am going to create a follow-up ticket (FL-24).

Sorry, but there is already a ticket [FL-23](https://linear.app/footlights/issue/FL-23/%E6%B8%B2%E6%9F%93%E6%97%B6%E7%BB%99%E6%AF%8F%E4%B8%AAyuan%E7%B4%A0%E5%A2%9E%E5%8A%A0%E7%BC%96%E5%8F%B7).

Reply:
Closing mine as a duplicate. I think it's sort of fair to add these TODO as a code comment